x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/expression/Alias.java

```diff
@@ -32,6 +32,7 @@ public final class Alias extends NamedExpression {
     public static final NamedWriteableRegistry.Entry ENTRY = new NamedWriteableRegistry.Entry(NamedExpression.class, "Alias", Alias::new);
 
     private final Expression child;
+    // TODO: unused, remove
```

Comment:
Why not do it now? Along with `qualifiedName()` and `qualifier()`?

Reply:
I guess I wanted to keep the blast radius smaller, but it turns out it's not so big. Removing!